pipelines/validator/README.md

```diff
@@ -0,0 +1,8 @@
+# validator
+A template showing how to use Popper with validations.
+In this case, Popper helps ensure that validations are enable by returning a gold when running the command: popper run.
+
+A gold status is returned when all validations were true at the validate.sh stage. A validation is true if it has the next format:
+> [true] Text about your validation.
+
+false otherwise.
```

Comment:
can we add more on what the validations are doing. A high-level description would be helpful.